content/20.treatments.md

```diff
@@ -302,8 +302,19 @@ All of these factors could potentially influence treatment outcome.
 Furthermore, the authors acknowledge that the effect of the drugs might be different in females and pediatric subjects, since these subjects were not the part of the study.
 
 **Summary.**
-In vitro evidence shows HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19.
-Multiple clinical studies have already been carried out to assess this possibility.
+In vitro evidence shows HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19. 
+It was recently discussed [@doi:10.1093/jac/dkaa114] that since EC90 of CQ in Vero E6 cells (6.90 μM) could be reached and well endured in rheumatoid arthritis patients, it could also be administered to COVID-19 patients.
+Additionally, it was proven to be effective against HIV [@doi:10.1016/0149-2918(95)80039-5] and chronic Hepatitis C infection [@doi:10.1002/jmv.24575].
```

Comment:
```suggestion
Additionally, HCQ was found to be effective in HIV [@doi:10/cq2hx9] and chronic Hepatitis C [@doi:10.1002/jmv.24575] patients.
```
Highlighting that these are not in vitro or model studies, and then the reason I changed your citation is just that there is a parenthesis and we have to go to shortdoi.org to find the comparable parenthesis-free citation -- you have been doing the citations totally correctly!